src/main/java/frc/robot/commands/IntakeCommand.java

```diff
@@ -0,0 +1,71 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.IntakeSubsystem;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+
+/**
+ * An example command that uses an example subsystem.
+ */
+public class IntakeCommand extends CommandBase {
+  @SuppressWarnings({ "PMD.UnusedPrivateField", "PMD.SingularField" })
+  private final IntakeSubsystem m_subsystem;
+  private RobotContainer m_container;
+
+  private boolean isLowered = false;
+
+  /**
+   * Creates a new ExampleCommand.
+   *
+   * @param subsystem The subsystem used by this command.
+   * @param container The instance(or mock) of RobotContainer used by this
+   *                  command.
+   * 
+   */
+  public IntakeCommand(IntakeSubsystem subsystem, RobotContainer container) {
+    m_subsystem = subsystem;
+    m_container = container;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(subsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if (m_container.mainController.getIntakePowerCells()) {
```

Comment:
I know that the example code from last year did this kind of things where a command would keep running and then poll the joystick buttons.  While this does work, the intended usage of the Button commands is to trigger on a button press, update the subsystem state, and then end.
We should talk about this.